packages/common/src/types.ts

```diff
@@ -176,6 +176,7 @@ export type DeviceType = 'desktop' | 'mobile' | 'tablet';
 export interface WalletModule {
   // The label of the wallet
   label: string;
+  dataAttribute: string;
```

Comment:
Not sure if it's good idea on having data attribute in the module. Couldn't label be used instead?

Reply:
switched to using slugified version of the label instead as discussed